libkbfs/block_retrieval_heap.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libkbfs
+
+type blockRetrievalHeap []*blockRetrieval
```

Comment:
Please add a `var _ heap.Interface = (*blockRetrievalHeap)(nil)` somewhere, so we can document it and enforce it within this file.